app/models/provider.rb

```diff
@@ -120,6 +120,10 @@ def current_accredited_courses
       .or(self.where(provider_code: Course.findable.select(:accrediting_provider_code)))
   end
 
+  scope :with_users, ->(users) do
+    joins(:users).merge(users)
+  end
```

Comment:
I think that this scope does a `join`, which potentially ends up returning multiple identical providers, is surprising. I'd leave it at `merge(users)`, or maybe just skip the scope altogether since even having `merge(users)` here is slightly odd as it requires an AREL `users`.

Reply:
removed inlined altogether